package org.apache.commons.lang3.builder;

import java.io.Serializable;

/* JADX INFO: compiled from: D8$$SyntheticClass */
/* JADX INFO: loaded from: classes5.dex */
public final /* synthetic */ class DiffBuilder$$ExternalSyntheticLambda15 implements DiffBuilder.SerializableSupplier, Serializable {
    public final /* synthetic */ byte f$0;

    public /* synthetic */ DiffBuilder$$ExternalSyntheticLambda15(byte b) {
        this.f$0 = b;
    }

    @Override // java.util.function.Supplier
    public final Object get() {
        return Byte.valueOf(this.f$0);
    }
}
